main.py

```diff
@@ -1,10 +1,43 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_info:
+        players = json.load(players_info)
+        for key, value in players.items():
+            person_email = value["email"]
+            person_bio = value["bio"]
+            if value["guild"] is not None:
+                Guild.objects.get_or_create(
```

Comment:
Store it into variable and use isntead of `guild_name`

Reply:
Do not retrieve data from `Guild` twice, do it once, avoid operation on line 19.